oas_worker/evaluate_devset.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env python
+""" evaluate_devset
+
+Evaluate OAS Devset:
+  * Keywords
+  * Transcripts (TBD)
+
+Author(s): flipsimon, datadonk23
+Date: 10.01.22
+"""
+
+import httpx
+import argparse
+import pandas as pd
+from datetime import datetime
+from string import punctuation
+
+from app.jobs.spacy_pipe import spacy_load
+from devset.evaluate_devset_utils import (
+    precision_recall_f1, mean_average_precision_k, log_results
+)
+
+
+OAS_URL = 'http://admin:password@localhost:8080/api/v1'
```

Comment:
Let's make this `os.environ.get('OAS_URL', 'http://admin:password@localhost:8080/api/v1')`